### PR TITLE
Add Client-side apps to glossary

### PR DESCRIPTION
**Pull request by martinkuba**
The Client-Side SIG is working on updates to the spec to distinguish client-side telemetry. We would like to add the term `Client-side` to the glossary, so that it is clear what it refers to and how it is distinguished from services.

### CODE REVIEW

File: content/en/docs/concepts/glossary.md
Comment (chalin) [inline content/en/docs/concepts/glossary.md:26]: Nit: please line wrap (at ~80 char) this paragraph.

**Review by chalin — COMMENTED** [review]
LGTM, though I'll let a SIG approver chime in.

**Review by mhennoch — APPROVED** [review]

**Review by mtwo — APPROVED** [review]

**Comment by chalin** [comment]
Thanks for the reviews

**Pull request merged**
